packages/sheets/src/index.ts

```diff
@@ -14,7 +14,6 @@
  * limitations under the License.
  */
 
-/* eslint-disable perfectionist/sort-named-exports */
```

Comment:
IHMO you should probably not to remove this line, because we have a special section index.ts files to exports all commands.

Reply:
cc @jikkai please take a look into this.

Reply:
This eslint ignore comment explicitly placed to maintain region comments for code folding and organization.